up42/jobcollection.py

```diff
@@ -72,5 +72,29 @@ def download_results(
             )
             out_filepaths[job.job_id] = out_filepaths_job
 
+        if merge:
+            merged_data_json = output_directory / "data.json"
+            with open(merged_data_json, "w") as dst:
+                out_features = []
+                for job_id in out_filepaths:
+                    all_files = out_filepaths[job_id]
+                    data_json = [d for d in all_files if d.endswith("data.json")][0]
```

Comment:
is there a possibility that the folder may contain a file, e.g. `metadata.json` which gets captured by this?

Reply:
It's a good question, I would guess it would get captured, and we don't want that. Changed to use the Path name property to make it more precise.